projects/env-semconv.md

```diff
@@ -0,0 +1,64 @@
+### Description
+
+This working group looks to identify and standardize observability for
+sustainability purposes with regards to infrastructure and applications. This
+would look at current tooling around sustainability metrics and standardize the
+meaning and approach for these metrics.
+
+Current efforts of implementations:
+
+- [Kepler](https://github.com/sustainable-computing-io/kepler)
+- [Scaphandre](https://github.com/hubblo-org/scaphandre)
+
+We want to have a unified metrics convention in order to better facilitate measuring sustainability metrics across different ecosystems.
+
+### Deliverables
+
+- Semantic conventions for sustainability metrics
+- Reference Prototypes metrics
+- Recommended practices for reporting on sustainability metrics 
+
+### Staffing / Help Wanted
+
+**Project Leads**
+- @spazzy757
+
+**Sponsoring Members**
+- @jpkrohling 
+- @pyohannes 
+
+**Engineers**
+- @gabibeyer
+- @bertysentry
+- @nikimanoledaki
+- @mkorbi
+- @rossf7
+- @marceloamaral
+
+### Meeting Times
+
+- Meeting Times: Mondays 4:00PM - 5:00PM CET
+- CNCF Slack [#otel-sustainability-semconv-sig](https://cloud-native.slack.com/archives/C06RTM63YD6) for collaboration
+
+### Timeline
+
+Q3 2024
+* Kickoff
+* Experimental semantic conventions
+Q4 2024
+* Prototype development
+1H 2025
+* Review and refinement
+* Declare sustainability semantic conventions as stable
+* Create Recommendations
```

Comment:
```suggestion
Q3 2024

* Kickoff
* Experimental semantic conventions

Q4 2024

* Prototype development

1H 2025

* Review and refinement
* Declare sustainability semantic conventions as stable
* Create Recommendations
```

Otherwise it's not formatted correctly.